acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -546,6 +545,6 @@ def check_puppet_non_blocking_response(identity, transaction_id, max_retries, qu
 end
 
 def get_mode(host, path)
-  ruby = Puppet::Acceptance::CommandUtils.ruby_command(host)
+  ruby = ruby_command(host)
```

Comment:
Does this work? When I use this command in beaker-puppet, I had to include `extend Beaker::DSL::InstallUtils::FOSSUtils` in the file

Reply:
It seemed to. I'm not exactly sure why.

Reply:
Also, why isn't jenkins testing?

Reply:
This totally works locally, even though I have no idea where the method is getting loaded. It's definitely calling the right one from beaker-puppet though, so I suppose this is good